src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write code here
+const fields = document.querySelectorAll('.field');
+const labelTemplate = document.createElement('label');
+
+labelTemplate.className = 'field-label';
+
+for (const field of fields) {
+  const lable = labelTemplate.cloneNode();
+  const input = field.lastElementChild;
+  const inputName = formatInputName(input.getAttribute('name'));
+
+  lable.textContent = inputName;
+  lable.htmlFor = input.id;
+
+  input.setAttribute('placeholder', capitalizInputName(inputName));
+  field.append(lable);
+}
+
+function formatInputName(str) {
+  const capitalIndex = [...str].findIndex(char => char.codePointAt(0) < 91);
```

Comment:
```suggestion
  const capitalIndex = str.findIndex(char => char.codePointAt(0) < 91);
```
Метод findIndex не мутує оригінальний масив. Так для чого тоді робити копію